src/components/LoanCards/SharedLoanCardController.vue

```diff
@@ -0,0 +1,307 @@
+<template>
+	<div>
+		<kv-classic-loan-card
+			:loan-id="loanId"
+			:loan="loan"
+			:custom-loan-details="customLoanDetails"
+			:use-full-width="useFullWidth"
+			:show-tags="showTags"
+			:category-page-name="categoryPageName"
+			:enable-five-dollars-notes="enableFiveDollarsNotes"
+			:large-card="largeCard"
+			:is-adding="isAdding"
+			:is-visitor="isVisitor"
+			:basket-items="basketItems"
+			:is-bookmarked="isBookmarked"
+			:kv-track-function="$kvTrackEvent"
+			:photo-path="PHOTO_PATH"
+			:show-view-loan="showViewLoan"
+			:external-links="true"
+			:route="currentRoute"
+			:user-balance="userBalance"
+			:get-cookie="getCookie"
+			:set-cookie="setCookie"
+			@toggle-bookmark="toggleBookmark"
+			@add-to-basket="addToBasket"
+		/>
+	</div>
+</template>
+
+<script>
+import * as Sentry from '@sentry/vue';
+import { gql } from '@apollo/client';
+import { setLendAmount, handleInvalidBasket, hasBasketExpired } from '@/util/basketUtils';
+import { readLoanFragment, watchLoanData } from '@/util/loanUtils';
+import bookmarkLoan from '@/util/bookmarkUtil';
+import logFormatter from '@/util/logFormatter';
+import { createIntersectionObserver } from '@/util/observerUtils';
+import percentRaisedMixin from '@/plugins/loan/percent-raised-mixin';
+import timeLeftMixin from '@/plugins/loan/time-left-mixin';
+import loanCardFieldsExtendedFragment from '@/graphql/fragments/loanCardFieldsExtended.graphql';
+import KvClassicLoanCard from '~/@kiva/kv-components/vue/KvClassicLoanCard';
+
+const PHOTO_PATH = 'https://www-kiva-org.freetls.fastly.net/img/';
+
+const loanQuery = gql`
+	${loanCardFieldsExtendedFragment}
+	query kcBasicLoanCard($basketId: String, $loanId: Int!) {
+	shop (basketId: $basketId) {
+		id
+		basket {
+			id
+			# for isInBasket
+			items {
+				values {
+					id
+				}
+			}
+		}
+	}
+	lend {
+		loan(id: $loanId) {
+			id
+			...loanCardFieldsExtended
+		}
+	}
+	my {
+		id
+		userAccount {
+			id
+		}
+	}
+}`;
+
+export default {
+	name: 'SharedLoanCardController',
+	props: {
+		loanId: {
+			type: Number,
+			required: true,
+		},
+		customLoanDetails: {
+			type: Boolean,
+			default: false
+		},
+		useFullWidth: {
+			type: Boolean,
+			default: false
+		},
+		showTags: {
+			type: Boolean,
+			default: false
+		},
+		categoryPageName: {
+			type: String,
+			default: '',
+		},
+		enableFiveDollarsNotes: {
+			type: Boolean,
+			default: false
+		},
+		enableMoreCta: {
+			type: Boolean,
+			default: false
+		},
+		largeCard: {
+			type: Boolean,
+			default: false
+		},
+		userBalance: {
```

Comment:
I know we have the balance available in the parent components in `ui`, but in order to allow this component to work similar to the CPS component, and because this component is already getting the balance in the query, we can remove `userBalance` as a prop and set it in this component.